tslib/src/azure/app-insights.ts

```diff
@@ -0,0 +1,34 @@
+import * as appInsights from 'applicationinsights'
+import config from '../config'
+
+const appInsightsHelper = {
+  startInsightsIfConfigured: (cloudRole: string) => {
+    if (config.ApplicationInsights.Key !== undefined) {
+      console.log('AppInsights Key found, initialising...')
```

Comment:
it can be useful to detect if its been initialised correctly, so i'll preserve it.  However, i will inject the logger interface, rather than use console.log

Reply:
scratch the above - i cant inject the context before the function is called!

have removed.